_posts/2020-01-15-weak-passwords.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Weak Passwords
+date: 2020-01-08 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+When I started using 1Password I manually imported the logins I remembered and changed the passwords foe the most important ones. The others were left with relatively weak passwords, but if they got compromised nothing bad would happen. A few days ago I became curious in finding those account and taking a look at them after almost a decade.

```

Comment:
The app doesn't let you select text and copy... So lame.

Rewrite the last sentence here.